runtime/testy/Ansible.py

```diff
@@ -116,7 +116,8 @@ def package(state):
                     for param, value in values[0].items():
                         param_value_pair = sensor.param_value.add()
                         param_value_pair.param = param
-                        param_value_pair.float_value = value[0]
+                        if value[0]:
```

Comment:
I would argue yes. 
It is possible that we do not have any sensors connected (even though its an incorrect configuration because PDB) 

In that scenario, we still want to send the state of the robot via studentCodeState